client/verta/verta/runtime.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+import json
+import threading
+import os
+
+
+_THREAD = threading.local()
+
+def init_thread_context() -> None:
```

Comment:
If you weren't intending to already, I'd recommend marking these `*_thread_context()` functions as private

```suggestion
def _init_thread_context() -> None:
```

as they are magic that we should discourage users from fiddling with or being concerned about, to my understanding.

Reply:
Updated to private functions as suggested.